src/Elastic.Markdown/Myst/Directives/DirectiveHtmlRenderer.cs

```diff
@@ -85,11 +84,14 @@ protected override void Write(HtmlRenderer renderer, DirectiveBlock directiveBlo
 
 	private void WriteImage(HtmlRenderer renderer, ImageBlock block)
 	{
-		var imageUrl =
-			block.ImageUrl != null &&
-			(block.ImageUrl.StartsWith("/_static") || block.ImageUrl.StartsWith("_static"))
-				? $"{block.Build.UrlPathPrefix}/{block.ImageUrl.TrimStart('/')}"
-				: block.ImageUrl;
+		var imageUrl = block.ImageUrl;
+		if (!string.IsNullOrEmpty(block.ImageUrl))
+		{
+			if (block.ImageUrl.StartsWith('/') || block.ImageUrl.StartsWith("_static"))
+				imageUrl = $"{block.Build.UrlPathPrefix}/{block.ImageUrl.TrimStart('/')}";
+			else
+				imageUrl =  block.Build.UrlPathPrefix + block.CurrentFile.DirectoryName!.Replace(block.Build.ConfigurationPath.DirectoryName!, "") + "/" + block.ImageUrl;
```

Comment:
Can we accomplish this using `Path.GetRelativePath()` ?

Reply:
Thanks for the hint!

https://github.com/elastic/docs-builder/pull/532/commits/73a3598737ff5af45c36c9b9a38bb85a521d8f56

Reply:
https://github.com/elastic/docs-builder/pull/532/commits/55b4b193ad3365c0d4699cd9ed705d948f550bf6